src/components/Header/Header.scss

```diff
@@ -0,0 +1,87 @@
+@import '../../variables.scss';
+@import '../../mixins.scss';
+
+.header {
+    width: 100%;
+    height: 300px;
+    position: relative;
+
+    .header_cover {
+        width: 100%;
+        height: 100%;
+        background-image: url(moviecover.jpeg);
+        filter: blur(2px);
+        z-index: -1;
+
+        @include absoluteToTopLeft
+    }
+
+    .header_cover--black {
+        width: 100%;
+        height: 100%;
+        background-color: rgba(0, 0, 0, 0.4);
+
+        @include absoluteToTopLeft
+    }
+
+    .header_logo {
+        // position: absolute;
+        // top: 20px;
+        // left: 60px;
+
+        @include absoluteByArgument(20px, unset, unset, 60px);
+
+        color: $magentaPink;
+
+        span {font-weight: bold;}
+    }
+
+    .header_addMovieButton {
+        // position: absolute;
+        // top: 20px;
+        // right: 60px;
+
+        @include absoluteByArgument(20px, 60px);
+
+        color: $magentaPink;
+        
+    }
+
+    .header_searchWrapper {
+        width: 80%;
+        height: 100%;
+        margin: auto;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+
+        h2 {
+                margin-top: 0;
+                text-transform: uppercase;
+                color: white;
+                font-weight: normal;
+            }
+
+        .header_searchWrapper_inputContainer {
```

Comment:
This hierarchy and inclusion is already represented by the naming itself, it is not neccessary to put the block directly inside the parent as well. No need to push specificity further in this case :)

For example:

```
// Preferred
.lorem {};
.lorem__ipsum {}

// Instead of this
.lorem {
  ...
  .lorem__ipsum {}
}
```